VK/VK/Controllers/FriendPhotoCollectionViewController.swift

```diff
@@ -54,18 +58,47 @@ final class FriendPhotoCollectionViewController: UICollectionViewController {
             withReuseIdentifier: Constants.photosCellIdentifier,
             for: indexPath
         ) as? PhotoCollectionViewCell else { return PhotoCollectionViewCell() }
-        cell.configure(photoNames, likeCount, false)
+        cell.configure(photoNames)
         return cell
     }
 
     // MARK: - Private Methods
 
     private func fetchFriends() {
-        networkService.fetchPhotos(friendID) { [weak self] photos, likes in
+        networkService.fetchPhotos(friendID) { [weak self] photo in
             guard let self = self else { return }
-            self.photoNames = photos
-            self.likeCount = likes
-            self.collectionView.reloadData()
+            switch photo {
+            case let .success(data):
+                let friendsDetail = data.friendDetail.friendPhotos.map(\.photos.last)
+                self.photoNames = friendsDetail.map { $0?.url ?? Constants.emptyString }
+                self.realmService.saveData(data.friendDetail.friendPhotos)
+                self.collectionView.reloadData()
+            case let .failure(error):
+                print(error)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
и лучше в алерте показывать ошибку